src/shared/components/query/QueryAndDownloadTabs.tsx

```diff
@@ -64,6 +65,8 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 	@autobind
 	onSelectTab(tabId:string)
 	{
+		this.store.forDownloadTab = tabId === DOWNLOAD
```

Comment:
this method should be @action because it sets multiple observable values

Reply:
Thanks! I should remember this, I updated my pr.